package-lock.json

```diff
@@ -0,0 +1,1902 @@
+{
```

Comment:
I strongly recommend _not_ committing a `package-lock.json` to this repo. It will lock the dependencies used by CI and loopback-connector-mysql devs while _users_ of loopback-connector-mysql will get newer/updated versions of the dependencies.

This means that CI will not be testing what users are using and may missing important bugs introduced by upstream dependencies.

Reply:
True, deleted 🙂 it's auto generated by my PR, now tests run successfully, thanks!